mars/services/scheduling/api.py

```diff
@@ -77,6 +79,10 @@ async def create_session(cls: Type[APIType],
             uid=SubtaskManagerActor.gen_uid(session_id)
         )
 
+        from .supervisor.autoscale import AutoscalerActor
+        autoscaler_ref = await mo.actor_ref(AutoscalerActor.default_uid(), address=address)
```

Comment:
Why not storing this actor ref as a member of hosting actor?

Reply:
What do you mean `hosting actor`?